### DISCUSSION TITLE
The example page fails to load

### DISCUSSION THREAD

**Discussion by jinyuanqing** [singerdmx/flutter-quill#842] [Web] [answered]
flutter3.0.1
**vscode tips:**
 EXCEPTION CAUGHT BY WIDGETS LIBRARY 
The following UnimplementedError was thrown building TextLine(dirty, dependencies:
[_QuillPressedKeysAccess], state: _TextLineState#c3c72):
Embeddable type "video" is not supported by default embed builder of QuillEditor. You must pass your
own builder function to embedBuilder property of QuillEditor or QuillField widgets.
The relevant error-causing widget was:
TextLine
**google chrome tips:**
Assertion failed:file:///xxxx/src/widgets/framework.dart:6243.12
renderObject.child--child is not true

**Comment by singerdmx** [comment, marked as answer]
This is fixed now
